diff --git a/bin/pycharm.vmoptions b/bin/pycharm.vmoptions
index 82e3538..f456445 100644
--- a/bin/pycharm.vmoptions
+++ b/bin/pycharm.vmoptions
@@ -1,7 +1,6 @@
 -server
 -Xms128m
 -Xmx512m
--XX:MaxPermSize=250m
 -XX:ReservedCodeCacheSize=240m
 -XX:+UseConcMarkSweepGC
 -XX:SoftRefLRUPolicyMSPerMB=50
diff --git a/bin/pycharm64.vmoptions b/bin/pycharm64.vmoptions
index 6c9f43d..ef57996 100644
--- a/bin/pycharm64.vmoptions
+++ b/bin/pycharm64.vmoptions
@@ -1,6 +1,5 @@
 -Xms128m
 -Xmx750m
--XX:MaxPermSize=350m
 -XX:ReservedCodeCacheSize=240m
 -XX:+UseConcMarkSweepGC
 -XX:SoftRefLRUPolicyMSPerMB=50
